docs/integrations/tax.mdx

```diff
@@ -153,6 +153,14 @@ To protect the shopper checkout experience from overly slow tax responses, tax e
   You may request a lower timeout threshold to provide a better service to merchants.
 </Callout>
 
+#### Handling additional inputs
+
+The Tax Provider API's request specification has been designed to include all necessary inputs required to perform a standard tax estimation. If your solution requires additional inputs beyond what the request model offers, then this additional data should be accessible on your system in a performant manner.
+
+In particular this means avoiding synchronous calls to external systems while preparing your tax estimate response. A recommended performant implementation strategy is to maintain a projection of external data on your system, performing updates asynchronously through background jobs or queues as required (e.g. [subscribing to BigCommerce webhooks](https://developer.bigcommerce.com/docs/integrations/webhooks)).
```

Comment:
```suggestion
Avoid synchronous calls to external systems when preparing your tax estimate response. We recommend you keep a copy of external data on your system and update it asynchronously through background jobs or queues (e.g., [subscribing to BigCommerce webhooks](https://developer.bigcommerce.com/docs/integrations/webhooks)).
```

Reply:
Fantastic, as always @bc-andreadao.

Only interesting thing here is the `e.g.,` update - on purpose? I'll update the others in this document if so - but keen to understand that update a little more 🙏

Reply:
Thanks @theromulans! Yes, there should be a comma after e.g. 
![Uploading comma.png…]()

Reply:
TIL, thank you @bc-andreadao!